src/docs/components/state.md

```diff
@@ -4,73 +4,277 @@ description: Use the State() for component's internal state
 url: /docs/state
 contributors:
   - jthoms1
+  - rwaskiewicz
 ---
 
-# State Decorator
+# State
 
-The `@State()` decorator can be used to manage internal data for a component. This means that a user cannot modify this data from outside the component, but the component can modify it however it sees fit. Any changes to a `@State()` property will cause the components `render` function to be called again.
+'State' is a general term that refers to the values and objects that are stored on a class for use now or in the future.
+The TypeScript class `CurrentTime` below stores the time that it was created, and is able to print it back to the user:
 
-## Example
+```ts
```

Comment:
I'm not sure this example adds much. I think your explanation here is very good, but may be better suited in the next section (The State Decorator). For me, it makes more sense to say here's the state decorator and here's why/when we use it all in one section.